internal/util/csiconfig.go

```diff
@@ -154,10 +154,55 @@ func GetMonsAndClusterID(ctx context.Context, clusterID string, checkClusterIDMa
 
 // GetClusterID fetches clusterID from given options map.
 func GetClusterID(options map[string]string) (string, error) {
-	clusterID, ok := options[clusterIDKey]
+	clusterID, ok := options[ClusterIDKey]
 	if !ok {
 		return "", ErrClusterIDNotSet
 	}
 
 	return clusterID, nil
 }
+
+// GetClusterIDFromMon will be called with a mon string to fetch
+// clusterID based on the passed in mon string. If passed in 'mon'
+// string has been found in the config the clusterID is returned,
+// else error.
+func GetClusterIDFromMon(pathToConfig, mon string) (string, error) {
+	clusterID, err := readClusterInfoWithMon(pathToConfig, mon)
+	if err != nil {
+		return "", err
+	}
+
+	if clusterID == "" {
```

Comment:
move this check to `readClusterInfoWithMon`, 

that way unit testcases for `readClusterInfoWithMon` will be sufficient.

Reply:
Done